src/react-components/banner/Banner.js

```diff
@@ -0,0 +1,321 @@
+import { ReactComponent as ArrowIcon } from "../icons/Arrow.svg";
+import warning_icon from "../../assets/images/warning_icon.png";
+import { TextInputField } from "../input/TextInputField";
+import { CheckboxInput } from "../input/CheckboxInput";
+import { FormattedMessage } from "react-intl";
+import cone from "../../assets/images/cone.png";
+
+const Banner = () => {
+  const [email, setEmail] = useState("");
+  const [confirm, setConfirm] = useState(true);
+  const [submitted, setSubmitted] = useState(false);
+  const [responseStatus, setResponseStatus] = useState(false);
+  const [isExpanded, setIsExpanded] = useState(false);
+  const { handleSubmit, register } = useForm({
+    defaultValues: {
+      email_format: "html"
+    }
+  });
+
+  /**
+   * On Form Error
+   */
+  const newsletterError = () => {
+    setResponseStatus(false);
+    setSubmitted(true);
+  };
+
+  /**
+   * On Form Success
+   */
+  const newsletterSuccess = () => {
+    setResponseStatus(true);
+    setSubmitted(true);
+  };
+
+  /**
+   * On XHR Load
+   * @param {Basket Response} resp
+   * @returns
+   */
+  const onload = resp => {
+    // Check Target Status
+    const status = resp.target.status;
+    if (status !== 200 || status > 300 || status < 200) {
+      newsletterError();
+      return;
+    }
+
+    // Check Response Status
+    let response = resp.target.response;
+    typeof response !== "object" ? (response = JSON.parse(response)) : "";
+    if (response.status !== "ok") {
+      newsletterError();
+      return;
+    }
+
+    // Do success stuff here..
+    newsletterSuccess();
+  };
+
+  const onSubmit = async data => {
+    // Note: this is a pattern that was suggesed by the basket creator
+    const url = "https://basket.mozilla.org/news/subscribe/";
+    const xhr = new XMLHttpRequest();
+    const params =
+      "email=" +
+      encodeURIComponent(data.email) +
+      "&newsletters=" +
+      "hubs" +
+      "&lang=" +
+      encodeURIComponent(navigator.languages[0]) +
+      "&source_url=" +
+      encodeURIComponent(window.location);
+
+    xhr.onload = onload;
+    xhr.open("POST", url, true);
+    xhr.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
+    xhr.setRequestHeader("X-Requested-With", "XMLHttpRequest");
+    xhr.timeout = 5000;
+    xhr.ontimeout = newsletterError;
+    xhr.responseType = "json";
+    xhr.send(params);
+  };
+
+  /**
+   * Emain input change
+   */
+  const onChangeEmail = useCallback(
+    e => {
+      setEmail(e.target.value);
+    },
+    [setEmail]
+  );
+
+  /**
+   * Checkbox Confirm
+   */
+  const onConfirm = () => {
+    setConfirm(state => !state);
+  };
+
+  /**
+   * Checkbox Label
+   * @returns JSX
+   */
+  const Label = () => {
+    return (
+      <>
+        I&apos;m okay with Mozilla handling my info as explained in this{" "}
+        <a href="https://github.com/mozilla/hubs/blob/master/PRIVACY.md" rel="noopener noreferrer" target="_blank">
```

Comment:
Maybe this should link to
https://www.mozilla.org/en-US/privacy/websites/
or 
https://www.mozilla.org/en-US/privacy/hubs/